dry_adiabatic_adjust/dadadj.F90

```diff
@@ -38,7 +38,8 @@ subroutine dadadj_init(dadadj_nlvdry, dadadj_niter, nz, errmsg, errflg)
 
     if (dadadj_nlvdry >= nz .or. dadadj_nlvdry < 0) then
        errflg = 1
-       write(errmsg,*) 'dadadj_init: dadadj_nlvdry=',dadadj_nlvdry,' but must be less than the number of vertical levels '
+       write(errmsg,*) 'dadadj_init: dadadj_nlvdry=',dadadj_nlvdry,' but must be less than the number of vertical levels ',&
+       '(',nz,'), and must be a positive integer.`
```

Comment:
Whoops, I think I had a typo for this particular suggestion:

```suggestion
       '(',nz,'), and must be a positive integer.'
```

Reply:
Done.